src/entityCreation/README.md

```diff
@@ -0,0 +1,141 @@
+# 🧱 Entity Management - Creation Flow
+
+This guide provides step-by-step instructions to manage entities in the SAAS platform across different roles and environments.
+
+---
+
+## 🔐 Auth Keys & Tokens
+
+### 🔸 Org Admin - QA
+
+| Key                     | Value                     |
+| ----------------------- | ------------------------- |
+| `internal-access-token` | `{internal-access-token}` |
+| `X-auth-token`          | `{Token}`                 |
+
+### 🔸 Super Admin - QA
+
+| Key                     | Value                     |
+| ----------------------- | ------------------------- |
+| `internal-access-token` | `{internal-access-token}` |
+| `X-auth-token`          | `{Token}`                 |
+| `admin-auth-token`      | `{admin-auth-token}`      |
+| `tenantId`              | `shikshagraha`            |
+| `orgId`                 | `blr`                     |
+
+---
+
+## 🌐 Origin URLs (Per Organization)
+
+| Organization | Origin URL                      |
+| ------------ | ------------------------------- |
+| Shikshalokam | `shikshalokam-qa.tekdinext.com` |
+| Shikshagraha | `shikshagrah-qa.tekdinext.com`  |
+| Super Admin  | `default-qa.tekdinext.com`      |
+
+---
+
+## 🔑 Login API
+
+<details>
+<summary>Login API</summary>
+
+```bash
+curl --location '{{baseURL}}/user/v1/account/login' \
+--header 'Content-Type: application/json' \
+--header 'origin: shikshalokam-qa.tekdinext.com' \
+--data-raw '{
+  "identifier": "email/phone",
+  "password": "password"
+}'
+```
+
+</details>
+
+---
+
+**NOTE**: If you are an Organization Admin, please ensure that the headers include the values specified under the Organization Admin column.
+If you are a Super Admin, please include the corresponding values from the Super Admin column in the request headers.
+
+## 🧱 Add Entity Type
+
+<details>
+<summary>Add Entity Type API</summary>
+
```

Comment:
@MallanagoudaB can we include a small description for each step. The idea is that some one who has no idea about this also should understand what they are doing once they go through this readme. Anyway we are putting an effort to create this.lets make it useful.